src/cli/cli.cpp

```diff
@@ -1251,6 +1251,42 @@ template <> otError Interpreter::Process<Cmd("ccm")>(Arg aArgs[])
     return ProcessEnableDisable(aArgs, otThreadSetCcmEnabled);
 }
 
+/**
+ * @note `OPENTHREAD_CONFIG_REFERENCE_DEVICE_ENABLE` is required for all `test` sub-commands.
+ */
```

Comment:
```suggestion
```

We can remove this, it isn't needed here, these notes should be in the block for each command that needs it.